crates/driver/src/tests/cases/internalization.rs

```diff
@@ -10,7 +10,9 @@ use crate::tests::{
 async fn valid_internalization() {
     let test = setup()
         .pool(ab_pool())
-        .order(ab_order().internalize())
+        .order(ab_order()
```

Comment:
nit: to make the diff smaller and slightly easier to review it would be nice if these changes that are no longer necessary could be reverted.
* formatting (not sure why clippy is seemingly fine with both versions)
* referring to `base_order` instead of `ab_order()`.